src/zmalloc.c

```diff
@@ -388,8 +404,16 @@ char *zstrdup(const char *s) {
 }
 
 size_t zmalloc_used_memory(void) {
-    size_t um = atomic_load_explicit(&used_memory, memory_order_relaxed);
-    return um;
+    /* Maybe become negative */
+    int64_t um = atomic_load_explicit(&used_memory, memory_order_relaxed);
+    return um < 0 ? 0 : um;
```

Comment:
How can it become negative? Only if we allocate in one thread and free in another?

Reply:
Yes, allocate in main thread and free in bio?